addons/website_slides/static/src/js/slides_embed.js

```diff
@@ -7,6 +7,7 @@
 $(function () {
 
     if ($('#PDFViewer') && $('#PDFViewerCanvas')) { // check if presentation only
+        var MIN_ZOOM=1, MAX_ZOOM=10, ZOOM_INCREMENT=.5;
```

Comment:
I'm not sure the <1 zoom is useful, it gives something like this:

![image](https://user-images.githubusercontent.com/64349/100981725-76fa5700-3547-11eb-850c-6faec3aa419a.png)

Reply:
Maybe if you resize your window? Sure it's not very useful but it doesn't hurt to have it.

Reply:
There is one thing that this makes more difficult, is to reset the zoom.  I added the possibility to zoom out via Ctrl+MouseWheel, so that you can do it very quickly - just one big finger roll.  But if the zoom will go down to negative, I have to then zoom back in to get back to the useful page-fit level.  So I think, if the zoom will let you get to negative, it also needs a reset button to get back quickly to the page-fit level.